pkg/reconciler/v1alpha1/serverlessservice/resources/services.go

```diff
@@ -0,0 +1,150 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package resources
+
+import (
+	"github.com/knative/pkg/kmeta"
+	"github.com/knative/serving/pkg/apis/networking/v1alpha1"
+	"github.com/knative/serving/pkg/reconciler/v1alpha1/serverlessservice/resources/names"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+const (
+	networkingGroup      = "networking.internal.knative.dev"
+	sksLabelKey          = networkingGroup + "/serverlessservice"
+	servicePortNameHTTP1 = "http"
```

Comment:
'http1' and 'h2c'  are used as the well known port names for Revisions. Is there a reason to change them here?

https://github.com/knative/serving/blob/master/docs/runtime-contract.md#protocols-and-ports

Reply:
In the end there should be one set of constants.
But those are the ports on the k8s service which are are different.

Reply:
Perhaps we can unify those as well. But I'd leave it as the final step.